keps/NNNN-kep-template/README.md

```diff
@@ -270,6 +266,55 @@ when drafting this test plan.
 [testing-guidelines]: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
 -->
 
+[ ] I/we understand the owners of the involved components may require updating
+existing tests to make this code solid enough prior committing changes necessary
+to implement this enhancement.
+
```

Comment:
Maybe a subheading here that reqs an explicit answer - sometimes the comment string gets a bit lost, this will ensure people fill out the below feedback section, so something like:
#### Prereqs: Updates to existing tests needed?

(it's a terrible phrase, I know feel free to make it better)

Reply:
Proposed "Prerequisite testing updates"

If you don't like it, I would be happy to change (agree the subheading is useful).